packages/xo-server-sdn-controller/src/index.js

```diff
@@ -73,6 +73,21 @@ async function fileExists(path) {
 // =============================================================================
 
 class SDNController extends EventEmitter {
+  /*
+  Attributes on created networks:
+  - `other_config`:
+    - `encapsulation`    : encapsulation protocol used for tunneling (either `gre` or `vxlan`)
+    - `pif_device`       : PIF on which the tunnels are created, must be physical and have an IP configuration
+    - `private_pool_wide`: `true` if the network is created (and so must be managed) by a SDN Controller
+    - `vni`              : VxLAN Network Identifier, it is used by OpenVSwitch to route traffic of different networks in a single tunnel
```

Comment:
These entries are not namespaced, is there a risk of collision in the future?

Reply:
I wanted to ask you that question.
I don't think there's a risk but I can add xo:xo-server:xo-server-sdn-controller namespace.

Reply:
Will that break existing objects?

No need to over namespace, `xo:sdn-controller:` should be more than enough.

Reply:
Done in a separate PR, i'll put this one as WiP waiting for rebase and update the doc accordingly.